backend/main/views.py

```diff
@@ -18,6 +19,12 @@ def index(request):
     conversation_list = models.Conversation.objects.all()
     return render(request, "main/index.html", locals())
 
+def get_conversations(request):
+    if request.method != "GET":
+        return HttpResponseNotAllowed(["GET"])
+
+    conversation_list = models.Conversation.objects.all()
+    return HttpResponse(serializers.serialize("json", conversation_list), content_type="application/json")
```

Comment:
You might be interested in that there is a `JsonResponse` which practically includes the content-type.

https://docs.djangoproject.com/en/4.2/ref/request-response/#jsonresponse-objects